### ISSUE TITLE
JAWS does not give any feedback on VSCode after switching back from a desktop window

### ISSUE THREAD

**Issue by venkateshpotluri** [Microsoft/vscode#54035]
Issue Type: <b>Bug</b>

JAWS does not give any feedback after switching away from VSCode to a non web window such as the desktop, and switching back to VSCode. The screen reader does not announce any control or element that is on screen. Moreover, on pressing the Numpad plus (the pc cursor key for JAWS), JAWS announces "PC cursor" instead of "virtual PC cursor". The temperary fix is to close and re-launch VSCode.

## Steps to Reproduce

1. with JAWS enabled, launch VSCode.
1. press tab to hear JAWS announce an element on the screen.
1. press the windows key + d to go to the desktop.
1. press alt+tab and switch to the VSCode window.
1. try navigating using tab, shift tab, the direction keys, or any other screen reader mechanism to 
navigate.

pressing the numpad plus will result in JAWS announcing "virtual PC cursor".

### Observed Behavior

JAWS does not give any feedback after switching back to the VSCode window. pressing the numpad plus will result in JAWS announcing "PC cursor" instead of "virtual PC cursor".

### Expected Behavior
JAWS should work as it was before switching to the desktop.

VS Code version: Code 1.25.0 (0f080e5267e829de46638128001aeb7ca2d6d50e, 2018-07-05T13:11:58.697Z)
OS version: Windows_NT x64 10.0.15063

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4790S CPU @ 3.20GHz (8 x 3193)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.90GB (6.19GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|yes|
|VM|0%|

</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
latex-workshop|Jam|5.6.1
vs-keybindings|ms-|0.2.0
debugger-for-chrome|msj|4.7.0

</details>

**Comment by isidorn** [comment]
Currently we do not support JAWS, only NVDA on windows and VoiceOver on mac.
